catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/DeleteOperations.java

```diff
@@ -260,6 +269,80 @@ private DeleteResponse performLocalDelete(DeleteRequest deleteRequest,
         return deleteResponse;
     }
 
+    //
+    // Private helper methods
+    //
+
+    private DeleteRequest rewriteRequestToAvoidHistoryConflicts(DeleteRequest deleteRequest)
+            throws UnsupportedQueryException {
+        final String attributeName = deleteRequest.getAttributeName();
+        if (Metacard.ID.equals(attributeName)) {
+            return deleteRequest;
+        }
+
+        Filter filter = deleteRequest.getAttributeValues()
+                .stream()
+                .map(Object::toString)
```

Comment:
Will this actually work for all possible unique identifiers?  It would be odd to use but there might be issues if they use unique dates, floats, location, and binary attributes in their requests.